README.md

```diff
@@ -34,12 +34,27 @@ Use root or package level commands to run specific package, e.g. web:
 
 [/packages/web] yarn start
 ```
+If you use macOS you should give permission for gradlew directory 
+```
+[/] chmod 755 packages/mobile/android/gradlew
+```
```

Comment:
I'd move this part below the android part, feels like it belongs under `You can run iOS simulator only on macOS`

Reply:
I have fixed this, check pls.